src/assets/axiosGet.jsx

```diff
@@ -8,24 +8,28 @@ const request = axios.create({
 export const fetchAllArticles = () => {
     return request.get(`/articles`)
     .then((response) => {
-        console.log("all articles", "\n" ,response)
         return response
     })
 }
 
 export const fetchSingleArticle = (article_id) => {
     return request.get(`/articles/${article_id}`)
     .then((response) => {
-        // console.log("single article", "\n" ,response)
         return response.data.response
     })
 }
 
 export const fetchArticleComments = (article_id) => {
     return request.get(`/articles/${article_id}/comments`)
     .then((response) => {
-        // console.log("article comments", "\n" ,response)
         const sortedComments = response.data.comments.sort((a, b) => b.votes - a.votes)
         return sortedComments
     })
 }
+
+export const incramentVote = (article_id, num) => {
+    return request.patch(`/articles/${article_id}`, {inc_votes: num})
+    .catch((err) => {
+        console.log(err)
+    })
```

Comment:
Could you be doing more here to handle if there is an error with the patch request ?